### ISSUE TITLE
Is the amd branch going to be merged?

### ISSUE THREAD

**Issue by gaming-hacker** [flame/blis#166]
all,
it this going to eventually be merged back into the master?

**Comment by fgvanzee** [comment]
@gaming-hacker Yes, I am working on the merge of `amd` back into `master` now, along with other improvements that will be included in the merge. I could be done later this week. Thanks for your patience.
